app/src/main/java/org/jellyfin/androidtv/ui/itemdetail/FullDetailsActivity.java

```diff
@@ -1260,6 +1267,19 @@ public void onClick(View v) {
                 more.show();
             }
         });
+        if (userPreferences.getValue().get(UserPreferences.Companion.getVideoPlayer()) == PreferredVideoPlayer.EXTERNAL) {
+            playWithButton = new TextUnderButton(this, R.drawable.ic_add, buttonSize, 3, "Play with", new View.OnClickListener() {
+                @Override
+                public void onClick(View view) {
+                    Toast.makeText(mApplication, "Play With Button", Toast.LENGTH_SHORT).show();
```

Comment:
This toast looks like it was intended for debugigng purposes, should probably remove it or replace it with a logger debug message.